pkg/config/config.go

```diff
@@ -654,19 +653,46 @@ func (c *config) Load(path string, fsys fs.FS) error {
 		}
 		c.Storage.Buckets[name] = bucket
 	}
-	for slug, function := range c.Functions {
+
+	// Load functions config
+	entries, err := fs.ReadDir(fsys, builder.FunctionsDir)
```

Comment:
We now have to scan the functionsDir in order to detect `deno.json(c)` files and populate the `ImportMap` accordingly.